coalib/misc/VersionCheck.py

```diff
@@ -0,0 +1,8 @@
+from pkg_resources import get_distribution
+
+
+def version_check(package_name, version):
```

Comment:
1. Rename this to `check_version`, that more describes an action
2. Docstring missing